test/token_rules/check_global_constraints.js

```diff
@@ -43,7 +43,6 @@ contract('TokenRules::checkGlobalConstraints', async () => {
                 ),
                 'Should revert as transfers "to" and "amount" arrays length '
                 + 'are not equal.',
-                '\'to\' and \'amount\' transfer arrays\' lengths are not equal.',
```

Comment:
Why this check has been removed?

Reply:
Same error as above:

1) Contract: TokenRules::checkGlobalConstraints
       Contract: Negative Tests
         Reverts if transfersTo and transfersAmount array lengths are not equal.:

      AssertionError [ERR_ASSERTION]: 
The contract should revert with:
	"'to' and 'amount' transfer arrays' lengths are not equal." 
instead received:
	"Returned error: VM Exception while processing transaction: revert"